lib/crash_course_agent.ex

```diff
@@ -0,0 +1,35 @@
+defmodule CrashCourseAgent do
+  use GenServer
+  @moduledoc """
+  Gen Server with Agent API
+  """
+  # Client
+  def start_link(state \\ %{}) do
```

Comment:
Use `@doc false` whenever you want to hide the function from the generated documentation. Use `@doc "string"` to make it appear nicely.